clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsMembershipManager.java

```diff
@@ -148,7 +149,7 @@ public int hashCode() {
 
     private final String groupId;
 
-    private String memberId = "";
+    private final String memberId = Uuid.randomUuid().toString();
```

Comment:
member id generated here, can be final since it's not going to change for the lifetime of the client